docs/drive-management.md

```diff
@@ -0,0 +1,111 @@
+# Drive management
+
+## Prerequisites
+* Working DirectPV plugin. To install the plugin, refer to the [plugin installation guide](./installation.md#directpv-plugin-installation).
+* Working DirectPV CSI driver in Kubernetes. To install the driver, refer to the [driver installation guide](./installation.md#directpv-csi-driver-installation).
+
+## Add drives
+Drives are added to DirectPV to provision volumes. This involves a two step process as shown below.
+
+1. Run `discover` command - `discover` command probes eligible drives from DirectPV nodes and stores drive information in a YAML file. You should carefully examine the YAML file and set the `select` field to `yes` or `no` value to indicate drive selection. `select` field is set to `yes` value by default. Below is an example of the `discover` command:
```

Comment:
Can we also mention about the `--drives` and `--nodes` selector flags here? They can review `drives.yaml` optionally if they want. Instead, the table that we show would be useful for the review here.

Reply:
Actually these flags are explained `command reference guide`. However reviewing `drives.yaml` by the user must be done even these flags are used. Because drive state might change at the time of fetching the information.